### ISSUE TITLE
Add support for Windows 10 style start menu, but with cascading menu support

### ISSUE THREAD

**Issue by DManowitz** [Open-Shell/Open-Shell-Menu#1851]
### Is your feature request related to a problem? Please describe.

No

### Describe the solution you'd like

I really like the idea of Open Shell, but there are some things about the Windows 10 style start menu that I like.  However, one thing I don't like about the Windows 10 start menu is the lack of support for cascading menus if there are multiple levels of folders in the Start Menu / Programs directory hierarchies.  What I'd like to see in Open Shell is support for a Windows 10 style start menu (i.e., mixing of application links and folders together alphabetically), but with added support for cascading menus, like the Windows 7 style start menu has.

### Area of issue

Start menu

**Comment by ge0rdi** [comment]
> mixing of application links and folders together alphabetically

Is this the only thing you are missing in current Windows 7 style?

**Comment by DManowitz** [comment]
Yes, that is the only thing I'd really want added now. I suppose it would
be easier to add that to Windows 7 style menus than to implement Windows 10
style menus, so if you want to add that functionality to the Windows 7
style for now, that's fine with me.

However, at some point in the future, if you want to implement this
functionality in a Windows 10 style, that would be great.

**Comment by ge0rdi** [comment]
> Yes, that is the only thing I'd really want added now.

It will be great to change description of this enhancement, or just create new one if you'd like to keep this one too.

> I suppose it would
be easier to add that to Windows 7 style menus than to implement Windows 10
style menus

What you want looks like adding new setting and changing sorting order based on the setting.
That is far simpler than implementing menu that imitates Windows 10 start menu.

**Comment by HAYDENCRAFT2008** [comment]
Use Explorer Patcher
